src/chrome/content/rules/Google.com_Subdomains.xml

```diff
@@ -86,7 +87,6 @@ Non-2xx HTTP code: http://suggestqueries.google.com/ (200) => https://suggestque
 	<target host="hostedtalkgadget.google.com" />
 	<target host="investor.google.com" />
 	<target host="ipv4.google.com" />
-	<target host="ipv6.google.com" />
```

Comment:
@Hainish does our Travis support IPv6 currently? Related travis-ci/travis-ci/issues/6082 and EFForg/https-everywhere/pull/11127

cc @koops76 @Bisaloo